internal/internal_worker.go

```diff
@@ -220,6 +221,24 @@ func newWorkflowWorkerInternal(
 	return newWorkflowTaskWorkerInternal(taskHandler, service, domain, params)
 }
 
+var localActivityChannelMap = make(map[string]*localActivityChannel)
```

Comment:
Why do we have global map? instead we could have this shared between workflow and local activity worker during initialization and do not have task list name at all.

Reply:
will remove